src/Controller/ExportController.php

```diff
@@ -44,6 +49,21 @@ public function __construct(TimesheetRepository $timesheet, ServiceExport $expor
     {
         $this->timesheetRepository = $timesheet;
         $this->export = $export;
+        $this->query = $this->createExportQueryWithExports();
+    }
+
+    /**
+     * @return ExportQuery
+     */
+    protected function createExportQueryWithExports()
```

Comment:
I am not completely happy with this method being in the controller.
Maybe it could be moved as factory function to the ServiceExport. Registering the Query as a service could work as well and then registering the renderer type in a compiler pass.

I'll give it another thought and maybe push a change later on.